side-products/merkl/integration-guide.md

```diff
@@ -20,65 +22,79 @@ The `pools` object also has for each incentivized pool details about the differe
 
 You can filter from the \`pools\` object to only display the pools of your choice (for instance pools with tokens in a subset and with an active distribution).
 
-## Display user rewards
+## Displaying user rewards
 
 The API lets you get for a user through the `rewardsPerToken` object how much of rewards can be claimed for a specific pool.
 
 This object maps each reward token to the total amount of rewards accumulated on the pool (it is also inclusive of the rewards that have already been claimed) as well as to the amount of unclaimed tokens on the pool. It also details where these tokens have been earned (Uniswap V3, Gamma, ...).
 
-## Claim user rewards
+## Claiming user rewards
 
-On top of displaying how much has been earned by a specific address, the API can also be used to build the payload of a claim transaction of a user which accumulated rewards.&#x20;
+On top of displaying how much has been earned by a specific address, the API can also be used to build the payload of a claim transaction of a user which accumulated rewards.
 
-When called for a specific user, it returns in a `transactionData` payload the list of tokens with for each a Merkle proof to use to claim the associated rewards.
+When called for a specific user, it returns in a `transactionData` payload with the list of `tokens` to claim, the `amounts`, and a Merkle `proof` required to claim each of them.
 
-Rewards are claimable per token: meaning that if you have accumulated rewards of several tokens, you may choose to only claim your rewards of one token type, but you may also choose to claim all your token rewards at once.
+_Rewards are claimable per token: meaning that if you have accumulated rewards of several tokens, you may choose to only claim your rewards of one token type, but you may also choose to claim all your token rewards at once._
 
 The contract on which rewards should be claimed is the `Distributor` contract which address can be found on [this docs](helpers.md#🧑‍💻-smart-contracts), or on the [Angle SDK](https://github.com/AngleProtocol/sdk).
```

Comment:
make sure that with the reorganization you haven't broken links -> here the link helpers.md may be broken